anima/env/mayaEnv/auxiliary.py

```diff
@@ -914,7 +937,40 @@ def run_post_publishers():
 
         # before running use the staging area to store the current version
         staging['version'] = version
-        run_publishers(type_name, publisher_type=POST_PUBLISHER_TYPE)
+
+        # show splash screen during post publish progress and lock maya
```

Comment:
I didn't like the idea of showing a JPG file. Are we going to prepare a JPG file for all environments? Please use native PySide2 widgets.